common/core/web/keyboard-processor/tests/cases/engine/context.js

```diff
@@ -786,73 +892,87 @@ describe('Engine - Context Matching', function() {
 
   describe('handles simple deadkey contexts', function() {
     it('for the most basic cases:  DEADKEY_TEST_1', function() {
-      runEngineRuleSet(DEADKEY_TEST_1, "Deadkeys");
+      runEngineRuleSet([DEADKEY_TEST_1], "Deadkeys");
```

Comment:
Yeah... this bit right here was kind of important.  The `runEngineRuleSet` method expects an array, but each individual test is merely an object... an object _without length_.

Compare against this part...

https://github.com/keymanapp/keyman/blob/09ad4ce9af5be5695f82d11c02a0b3bf488a959e/common/core/web/keyboard-processor/tests/cases/engine/context.js#L26-L29

As a result, the `for`-loop was instantly shortcut, bypassing the actual test!  No bueno.  At least they'll _actually_ be running now.